lib/content-services/src/lib/tag/tags-creator/tags-creator.component.html

```diff
@@ -38,9 +39,10 @@
     class="adf-existing-tags-panel"
     *ngIf="existingTagsPanelVisible">
     <span *ngIf="!spinnerVisible || existingTags"
-        class="adf-create-tag-label"
-        (click)="addTag()"
-        [hidden]="tagNameControl.invalid || typing">
+          class="adf-create-tag-label"
+          (click)="addTag()"
+          tabindex="0" (keyup.enter)="addTag()"
```

Comment:
can you move (keyup.enter)="addTag()" to next line as all attributes were in column?